### ISSUE TITLE
Update `peerDependencies` to include React 18

### ISSUE THREAD

**Issue by Luukuton** [markusenglund/react-switch#107]
React 18 is now available: https://reactjs.org/blog/2022/03/29/react-v18.html

Trying to use React 18 together with react-switch v6.0.0 (latest version as the time of writing) leads to a dependency conflict:

```bash
 ERROR ➜  npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: mangatsu-web@0.3.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.3.0 || ^16.0.0 || ^17.0.0" from react-switch@6.0.0
npm ERR! node_modules/react-switch
npm ERR!   react-switch@"^6.0.0" from the root project
```

The culprit seems to be the peerDependencies defined [here](https://github.com/markusenglund/react-switch/blob/b5cd62f05794698b305e351fe544b8d638155aba/package.json#L32):

```json
  "peerDependencies": {
    "react": "^15.3.0 || ^16.0.0 || ^17.0.0",
    "react-dom": "^15.3.0 || ^16.0.0 || ^17.0.0"
  },
```

Would it be possible to publish a new version which allows react-switch to be used with React 18 too?

**Comment by coditori** [comment]
Hi is there any update?

**Comment by markusenglund** [comment]
Will try to get an update shipped this weekend.

**Comment by dnsnx** [comment]
Any updates to this issue? Would appriciate this very much. Thank you in advance!

**Comment by markusenglund** [comment]
Fixed in 6.1.0, sorry for the delay.
